src/github.com/matrix-org/dendrite/clientapi/routing/login.go

```diff
@@ -50,25 +64,112 @@ type loginResponse struct {
 	DeviceID    string                       `json:"device_id"`
 }
 
-func passwordLogin() loginFlows {
+func defaultPasswordLogin() loginFlows {
 	f := loginFlows{}
-	s := flow{"m.login.password", []string{"m.login.password"}}
+	s := flow{string(PasswordBased), []string{string(PasswordBased)}}
```

Comment:
Use the types in [logintypes.go](https://github.com/matrix-org/dendrite/blob/master/src/github.com/matrix-org/dendrite/clientapi/auth/authtypes/logintypes.go).